rmf_site_editor/src/site/model.rs

```diff
@@ -783,25 +786,42 @@ pub fn update_model_instances<T: Component + Default + Clone>(
     }
 }
 
+/// Unique UUID to identify issue of duplicated lift names
+pub const ORPHAN_MODEL_INSTANCE_ISSUE_UUID: Uuid =
+    Uuid::from_u128(0x4e98ce0bc28e4fe528cb0a028f4d5c08u128);
+
 pub fn assign_orphan_model_instances_to_level(
```

Comment:
It sounds like this function name is not accurate anymore after refactoring, it doesn't seem to assign instances to levels but instead report issues with orphan models

Reply:
The function attempts to add any orphan model instances to the current selected level first, and only reports an issue if a current level is not available. I initially thought this would be easier for users to recover from an orphan model instance, but now that you pointed it out I think it'd be better to simply raise an issue and have them respawn the model instances on their desired level. Updated in [a6d4139](https://github.com/open-rmf/rmf_site/pull/260/commits/a6d4139219d7eabae316a7946928e954c0163db0)